tests/core/create-user/task.yaml

```diff
@@ -1,8 +1,7 @@
 summary: Ensure that snap create-user works in ubuntu-core
 # FIXME: combine this test with tests/core/create-user-2?
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*]
+systems: [ubuntu-core-*]
```

Comment:
this line is now redundant, only ubuntu-core-* systems are available for the core suite

```suggestion
```

Reply:
done